index.html

```diff
@@ -738,18 +740,39 @@ <h3>Introduction</h3>
 					SVG images are not supported in the [=spine=] but can be embedded in XHTML documents.</p>
 			</section>
 
-			<section id="html-req">
-				<h3>Requirements</h3>
+			<section id="xhtml">
+				<h3>XHTML</h3>
 
-				<p>An [=eBraille content document=]:</p>
+				<p>An [=eBraille content document=] MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content
+						document</a> as defined [[epub-33]] with additional constraints as defined in the following
+					subsections.</p>
 
-				<ul>
-					<li>MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content document</a> [[epub-33]].</li>
-					<li>MUST NOT contain features specified in <a href="#html-no-support"></a>.</li>
-				</ul>
+				<section id="xhtml-char-encoding">
+					<h4>Character encoding</h4>
 
-				<section id="html-no-support">
-					<h3>Unsupported features</h3>
+					<p>[=eBraille content documents=] are required to provide their text content in 6- or 8-dot braille
+						characters. eBraille is not meant as a delivery format for ASCII braille or similar braille
+						encodings that require a translation to 6- or 8-dot braille characters.</p>
```

Comment:
If you put it this way, it seems ASCII braille is something in between that requires some processing that Unicode braille does not require. But to me Unicode braille and various ASCII braille variants are just different encodings of braille. They're on the same level. (But of course we prefer to follow the Unicode standard, and from the Unicode perspective, ASCII braille is odd).

Reply:
Ya, good point. I didn't mean it as translation to the unicode characters but the reading system has to know how to map the ascii braille to the correct display. I'll see if I can improve on it.